cogs/cipher.py

```diff
@@ -0,0 +1,111 @@
+import base64
+import hashlib
+import string
+
+from discord.ext import commands
+
+from assets.context import Command, argument, example, usage_info
+
+
+class Ciphers(commands.Cog):
+    character_set = [string.ascii_lowercase, string.ascii_uppercase, string.digits]
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_check(self, ctx):
+        return self.bot.is_ready():
+
+    @commands.command(cls=Command,
+                      description="Shifts the letters within a message",
+                      syntax=[argument(True, "Shift defaults to 3"), argument(True, "Message")],
+                      usage_information=[
+                          usage_info("", "If a shift value is not found at the start "
```

Comment:
Lines too long, of course, in addition to it not being written the canonical way